README.md

```diff
@@ -1,12 +1,10 @@
 # Accessible autocomplete
 
----
-
-**We cannot currently respond to issues or pull requests.**
+The accessible autocomplete is a component that helps users choose answers from a list you provide. You can also use it to standardise user input.
 
-You can still raise issues or pull requests if you want to. As soon as we’re able to, we will prioritise dealing with any bugs that have been raised by people in the UK public sector. We cannot prioritise adding new features.
+Do not use the accessible autocomplete for services where users have to provide their country or territory.
```

Comment:
'Do not use' feels a little over the top here. There might be valid reasons why the country and territory autocomplete doesn't meet a users needs, like the fact it relies on jQuery.

How about…

> If you're asking users to provide their country or territory, the [govuk-country-and-territory-autocomplete](https://github.com/alphagov/govuk-country-and-territory-autocomplete/blob/main/README.md) may be more appropriate.

Reply:
Ahh okay great, @EoinShaughnessy and I tried to see if there were other reasons but did not come up with anything.

Reply:
Looks great, thanks both 👍🏻